assets/css/style.css

```diff
@@ -124,6 +136,202 @@ h4.slogan {
   background: transparent;
 }
 
+.title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 30px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  padding-top: 15px;
+}
+
+.title-project {
+  color: #3a4a42;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 30px;
+  line-height: 44px;
+  text-align: center;
+  padding-top: 15px;
+}
+
+.about-me {
+  display: grid;
+  background-image:
+    url("../images/maskgroup2.svg"),
+    url("../images/maskgroup1.svg");
+  background-position:
+    0 90%,
+    100% 0;
+  background-repeat: no-repeat;
+}
+
+h3 {
+  margin-top: 12px;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 11px;
+  color: #172b4d;
+  font-family: Inter, sans-serif;
+  line-height: 44px;
+}
+
+#my-recent-work {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+}
+
+.indicator {
+  position: static;
+  width: 48px;
+  height: 4px;
+  margin: 0 auto;
+  background-color: #36b37e;
+  margin-bottom: 50px;
+}
+
+.img-patholder {
+  width: 327px;
+  height: 574px;
+  border: 1px solid #e1e1e1;
+  background: #fff;
+  border-radius: 8px;
+  margin: 10px auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+}
+
+.img-box {
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+  height: 340px;
+  width: 100%;
+  background: #ebf0ee;
+}
+
+.buttons {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  padding: 5px;
+  grid-gap: 5px;
+  margin-top: 10px;
+  margin-bottom: 5px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.buttons2 {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  padding: 5px;
+  grid-gap: 5px;
+  margin-left: 8px;
+  margin-right: 8px;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.buttons3 {
+  display: grid;
+  grid-template-columns: auto auto;
+  justify-content: center;
+  padding: 5px;
+  grid-gap: 5px;
+  margin-top: 0;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.buttons > li,
+.buttons2 > li,
+.buttons3 > li {
+  background-color: #ebf0ee;
+  border: none;
+  padding: 5px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  text-align: center;
+  border-radius: 5px;
+  font-size: 12px;
+  font-weight: 600;
+  font-family: Inter, sans-serif;
+  list-style-type: none;
+}
+
+.see-project {
+  display: flex;
+  justify-content: space-evenly;
+  margin-bottom: 30px;
+}
+
+.see-project button {
+  background-color: #36b37f;
+  border-radius: 5px;
+  border: none;
+  padding: 20px;
+  font-size: 18px;
+  color: #fff;
+  margin-top: 15px;
+}
+
+.get-my-resume button {
+  background-color: #36b37f;
+  border: none;
+  padding: 20px;
+  font-size: 18px;
+  color: #fff;
+  margin-top: 15px;
+  margin-bottom: 50px;
+  border-radius: 5px;
+}

```

Comment:
See project buttons and Get my resume button should follow the follow the button styling: Active, Hover and Pressed (see design)
![image](https://user-images.githubusercontent.com/83596135/153723609-471e687b-7acb-4812-9d66-29f4b9026a98.png)

- [ ] Kindly I ask to implement the styling to all your buttons as shown in the design